OpenDreamRuntime/DreamMapManager.cs

```diff
@@ -63,7 +63,7 @@ public sealed class Cell {
 
         public void Initialize() {
             _appearanceSystem = _entitySystemManager.GetEntitySystem<ServerAppearanceSystem>();
-            _mapManager.CreateNewMapEntity(MapId.Nullspace);
+            //_mapManager.CreateNewMapEntity(MapId.Nullspace);
```

Comment:
Is creating a nullspace map no longer necessary? If not then just remove the line.

Reply:
It is not only not necessary, but there's a `assert()` making sure you don't.